app/models/street/StreetEdgeTable.scala

```diff
@@ -374,15 +374,41 @@ object StreetEdgeTable {
         |       street_edge.y2,
         |       street_edge.way_type,
         |       street_edge.deleted,
-        |       street_edge.timestamp
+        |       street_edge.timestamp,
+        |       audit_task.completed
         |FROM street_edge
         |INNER JOIN audit_task ON street_edge.street_edge_id = audit_task.street_edge_id
         |WHERE street_edge.deleted = FALSE
         |    AND ST_Intersects(street_edge.geom, ST_MakeEnvelope(?, ?, ?, ?, 4326))
         |    AND audit_task.completed = TRUE""".stripMargin
     )
 
-    val edges: List[StreetEdge] = selectEdgeQuery((minLng, minLat, maxLng, maxLat)).list
+    val edges: List[(StreetEdge, Boolean)] = selectEdgeQuery((minLng, minLat, maxLng, maxLat)).list
+    edges
+  }
+
+  def selectStreetsWithin(minLat: Double, minLng: Double, maxLat: Double, maxLng: Double): List[(StreetEdge, Boolean)] = db.withSession { implicit session =>
```

Comment:
Can you add comments wherever we use the unnamed Boolean to indicate if a street was audited? Since it's unnamed, I think it's particularly important to add comments for it! Alternatively, you could create a new `case class` that contains the minimal amount of info that we need from the `street_edge` table along with the `audited` boolean!